stripe-patrons-data/src/main/scala/com/gu/patrons/AWSAsyncHandler.scala

```diff
@@ -0,0 +1,53 @@
+package com.gu.patrons
```

Comment:
moved/copied out of module-aws (so that the v1 dependencies don't get pulled in transitively everywhere)